widgetry/src/lib.rs

```diff
@@ -93,6 +93,16 @@ mod backend {
     pub use crate::backend_glow::*;
 }
 
+/// Like [`std::include_bytes!`], but also returns its argument, the relative path to the bytes
+///
+/// returns a `(path, bytes): (&str, &[u8])` tuple
```

Comment:
This is so we can write:
```
builder.image_bytes(include_labeled_bytes!("path/to/my/file.svg"))
```

Instead of:
```
builder.image_bytes((include_bytes!("path/to/my/file.svg"), "path/to/my/file.svg"))
```

The "label" is used as the cache key.